frontends/web/src/containers/SubmitInterface.js

```diff
@@ -63,6 +63,8 @@ const FileUpload = ({ values, filename, setFieldValue, disabled }) => {
 const SubmitInterface = (props) => {
   const { get, response } = useFetch();
   const context = useContext(UserContext);
+  const [sendCallSubmited, setSendCallSubmited] = useState(false);
+  const [sendCallDataSets, setSendCallDataSets] = useState(false);
```

Comment:
This should be 
```suggestion
  const [sendCallDatasets, setSendCallDatasets] = useState(false);
```
and should be changed throughout the PR.